### DISCUSSION TITLE
I have some internal transactions that cannot be indexed.

### DISCUSSION THREAD

**Discussion by qianyiwangmeng** [blockscout/blockscout#9481] [Q&A]
it is due to aa smart account.  How can I index such aa internal transactions? can the user-ops-indxer-server solve my problem？

**Comment by ekawibisono** [comment]
hello, can you provide detail about your problem ?

**Reply by qianyiwangmeng** [reply]
I am testing on L2, Polygon ZK-EVM.

1.When there are no AA transactions, indexing internal transactions works fine.
2.Once there is an AA transaction in the node, fetching internal transactions for indexing stops working.
3.Even if I start a new backend to index, it even at 0% directly.
![image](https://github.com/blockscout/blockscout/assets/94897566/a630cd43-811a-4580-9ef3-3e446dbf7cdd)

**Reply by ekawibisono** [reply]
can you check log on backend ? and screenshot ?

**Reply by qianyiwangmeng** [reply]
I think the key issue lies with AA transactions. In my previous tests, internal transactions could be successfully retrieved and were at 100% when there were no AA transactions on other chain. Is it that backend don't support EIP-4337?
![image](https://github.com/blockscout/blockscout/assets/94897566/47c26e1e-74d4-4c7a-befc-3c4faee209d7)

**Reply by ekawibisono** [reply]
can you show command run geth ?

**Reply by qianyiwangmeng** [reply]
I test in polygon zkevm

**Reply by qianyiwangmeng** [reply]
What specific information do I need to provide?
